crates/nu-command/src/platform/clear.rs

```diff
@@ -57,8 +61,8 @@ impl Command for Clear {
                 result: None,
             },
             Example {
-                description: "Clear the terminal and its scroll-back history",
-                example: "clear --all",
+                description: "Clear the terminal but not its scrollback history",
+                example: "clear -k",
```

Comment:
We generally like to stay verbose in examples.
```suggestion
                example: "clear --keep-scrollback",
```